libpod/container_internal_linux.go

```diff
@@ -1426,8 +1427,24 @@ func (c *Container) makeBindMounts() error {
 	// Make .containerenv
 	// Empty file, so no need to recreate if it exists
 	if _, ok := c.state.BindMounts["/run/.containerenv"]; !ok {
+		isRootless := 0
+		if rootless.IsRootless() {
+			isRootless = 1
+		}
+		imageID, imageName := c.Image()
+
 		// Empty string for now, but we may consider populating this later
-		containerenvPath, err := c.writeStringToRundir(".containerenv", "")
+		// Populate the .containerenv with container information
+		containerenv := fmt.Sprintf(`\
+engine="podman-%s"
+name=%q
+id=%q
```

Comment:
Exposing ID by default might be potentially dangerous. Could this be optional?

Reply:
Please explain how container ID is risky? 

We can add something to containers.conf to disable this, but I am not sure it is worth the effort.

Reply:
An attacker might want to know the container ID to determine the path of the attacker-crafted malicious file in the container rootfs.

Such path would be useful for the attacker when the attacker somehow managed to gain write access to e.g. /sys/kernel/uevent_helper.